src/Sylius/Bundle/PromotionBundle/Resources/translations/flashes.en.yml

```diff
@@ -4,3 +4,5 @@
 sylius:
     promotion_coupon:
         generate: 'Promotion coupons have been successfully generated.'
+    catalog_promotion:
+        remove: 'Removing of catalog promotion has been ordered. This process can take a while depending on the number of products.'
```

Comment:
```suggestion
        remove: 'Removing of catalog promotion has been requested. This process can take a while depending on the number of related products.'
```

Reply:
How about "affected" instead of "related"?

Reply:
In fact, both options do not fully reflect an action as we process the entire set of variants in the shop.
https://github.com/Sylius/Sylius/blob/499c769dbfb0113c592eb6b9cdff6095ef0d328e/src/Sylius/Bundle/CoreBundle/CatalogPromotion/Listener/CatalogPromotionEndedListener.php#L42

Reply:
I'd still be for adding the `affected` just to be more explicit about the context.
As it is now, someone might think that the time extends based on all the products available.